### PR TITLE
Fix pagination on live tv pages

### PR DESCRIPTION
**Pull request by thornbill**
**Changes**
Fixes pagination for live tv pages. (Requires unstable build of server to display the right start index.)

**Issues**
N/A

### CODE REVIEW

File: src/controllers/list.js [hunk 1]
```diff
@@ -15,12 +15,12 @@ import { appRouter } from '../components/appRouter';
 
 /* eslint-disable indent */
 
-    function getInitialLiveTvQuery(instance, params) {
+    function getInitialLiveTvQuery(instance, params, startIndex = 0, limit = 300) {
```
Comment (dmitrylyzo) [inline src/controllers/list.js:18]: iirc, the default value doesn't work for passed `null`.
In calls like this (play/queue/shuffle):
https://github.com/jellyfin/jellyfin-web/blob/64b3fc80dcbaa93c080b5ca7b10aac8c3a74ed35/src/controllers/list.js#L739
But this probably won't screw anything up if the server replaces `null` with `0` on its side.
Also, this is not related to the original idea of PR.

**Review by dmitrylyzo — APPROVED** [review]
Can't test, but LGTM

File: src/controllers/list.js (hunk 1 above)
Comment (thornbill) [reply src/controllers/list.js:18]: Good call. I think it is better to explicitly pass `0` for those. I went ahead and made the change here since it is a tiny change. :slightly_smiling_face:

**Review by dmitrylyzo — APPROVED** [review]
